### PR TITLE
Add translations to subscriptions module

### PR DESCRIPTION
**Pull request by lmac012**
Task/Issue URL: https://app.asana.com/0/1205648422731273/1208917047742015/f

### Description

This PR add translations for string resources in subscriptions-impl module.

### Steps to test this PR

QA-optional

### UI changes

No UI changes other than translations.

### CODE REVIEW

**Comment by lmac012** [comment]
* **#5352** <a href="https://app.graphite.dev/github/pr/duckduckgo/Android/5352?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈 <a href="https://app.graphite.dev/github/pr/duckduckgo/Android/5352?utm_source=stack-comment-view-in-graphite" target="_blank">(View in Graphite)</a>
* `develop`

<h2></h2>

This stack of pull requests is managed by <a href="https://graphite.dev?utm-source=stack-comment"><b>Graphite</b></a>. Learn more about <a href="https://stacking.dev/?utm_source=stack-comment">stacking</a>.

File: app/src/main/res/values-fr/strings.xml [hunk 1]
```diff
@@ -156,9 +158,15 @@
     <string name="settingsWebTrackingProtectionTitle">Protection contre le pistage sur le Web</string>
     <string name="settingsWebTrackingProtectionDescription">Activation par défaut</string>
     <string name="settingsHeadingSettings">Paramètres</string>
+    <string name="settingsHeadingMainSettings" tools:ignore="MissingTranslation">Réglages principaux</string>
```
Comment (karlenDimla) [inline app/src/main/res/values-fr/strings.xml:161]: @lmac012  do you know why we have `tools:ignore=“MissingTranslation” `for all the new translations in this PR?

File: autofill/autofill-impl/src/main/res/values-de/strings-autofill-impl.xml [hunk 2]
```diff
@@ -258,6 +258,6 @@
     <string name="importPasswordsProcessingResultDialogDoneButtonText">Verstanden</string>
     <string name="importPasswordsProcessingResultDialogResultImportFailed">Passwortimport fehlgeschlagen</string>
     <string name="importPasswordsProcessingResultDialogResultDuplicatesSkipped"><![CDATA[Übersprungen (doppelt oder ungültig): <b>%1$d</b>]]></string>
-    <string name="importPasswordsProcessingResultDialogResultPasswordsImported"><![CDATA[Passwörter importiert: <b>%1$d</b>]]></string>
+    <string name="importPasswordsProcessingResultDialogResultPasswordsImported"><![CDATA[Passwörter importiert: ** **<b>%1$d</b>]]></string>
```
Comment (karlenDimla) [inline autofill/autofill-impl/src/main/res/values-de/strings-autofill-impl.xml:261]: Seems like a rogue change.

**Review by karlenDimla — APPROVED** [review]

File: app/src/main/res/values-fr/strings.xml (hunk 1 above)
Comment (mikescamell) [reply app/src/main/res/values-fr/strings.xml:161]: They were meant to be translated at some point, but I misunderstood how things work with the translations. These should all be behind the feature flag.

**Pull request merged**